source/reference/methods.txt

```diff
@@ -356,6 +356,62 @@ Connection Methods
 
      - Description
 
+   * - :method:`Mongo()`
+
+     - JavaScript constructor to instantiate a database connection from
+       the ``mongo`` shell or from a JavaScript file.
+
+       The :method:`Mongo()` method takes a parameter called
```

Comment:
Note that, like in the old shell, it takes two arguments -- the first one is still the connection string

Reply:
Updated.